basyx.common/basyx.filerepository-backend/src/test/java/org/eclipse/digitaltwin/basyx/core/filerepository/backend/FileRepositoryTestSuite.java

```diff
@@ -149,5 +129,27 @@ public void doesNotExists() {
 		
 		assertFalse(fileNotExists);
 	}
+	
+	protected abstract FileRepository getFileRepository();
+	
+	protected FileMetadata createDummyFile() {
+		String name = "test";
+		String contentType = "txt";
+		String contentInput ="this is test data";
+		
+		InputStream content = new ByteArrayInputStream(contentInput.getBytes());
+		
+		return new FileMetadata(name, contentType, content);
+	}
+
+	protected String saveTestFile(FileRepository fileRepo, FileMetadata testFileMetadata) {
+		String filePath = testFileMetadata.getFileName();
+		
+		if(fileRepo.exists(testFileMetadata.getFileName())) {
```

Comment:
No need for these curly braces because there is only a single statement inside this if block.